central/globaldb/v2backuprestore/formats/postgresv1/format.go

```diff
@@ -10,6 +12,11 @@ func init() {
 	formats.MustRegisterNewFormat(
 		"postgresv1",
 		common.NewFileHandler(backup.PostgresFileName, false, restorePostgresDB),
-		// TODO: ROX-10087 deal with certs
+		common.NewFileHandler(path.Join(backup.KeysBaseFolder, backup.CaCertPem), true, formats.Discard),
```

Comment:
So all these files are thrown to discard.  Am I reading that correctly in that they are simply ignored during. the restore process?